src/settings/moly_server_settings.rs

```diff
@@ -1,11 +1,6 @@
 use makepad_code_editor::code_view::CodeViewWidgetExt;
 use makepad_widgets::*;
 
```

Comment:
This file will actually change drastically once we move the MolyServer related features into it's own view. 
these changes are just placeholders, as now the connection settings are handled in the providers dashboard.